test/minikube/minikube_domain_resync_test.go

```diff
@@ -70,55 +69,6 @@ func verifyProcessLabels(t *testing.T, namespaceName string, adminPod string) (a
 	return archiveVolumeClaims
 }
 
-func verifyLoadBalancer(t *testing.T, namespaceName string, adminPod string, deploymentOptions map[string]string) {
```

Comment:
This function is moved to `test/minikube/verify_utility.go` file as it is now reused between files with different build tags.